andela/travel_tool_front
# #160801033 Update Requester Picture on Modal on Approval Page

#### What does this PR do?
Update requester picture on modal on approval page

#### Description of Task to be completed?
- Update requester picture on modal on approval page

#### How should this be manually tested?
- Clone the repo - `git clone https://github.com/andela/travel_tool_front.git`
- Change into the project directory
- Run the server `make start`
- Login and navigate to the approval page
- Click on any request details on the page
- The picture of the request should be displayed.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#160801033](https://www.pivotaltracker.com/story/show/160801033)

#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2018-09-27 at 3 02 03 pm" src="https://user-images.githubusercontent.com/28366272/46152230-834c4080-c268-11e8-8210-7932127a7d86.png">


#### Questions:
N/A
